### ISSUE TITLE
Crow Translate is not working in Pop-up Window Translation mode

### ISSUE THREAD

**Issue by leonid1812** [crow-translate/crow-translate#663]
**Bug description**

The Crow Translate application is not working in **Pop-up Window** _Translation mode_. The application does not respond to HotKeys (Ctrl+Alt+E). However, everything works as if the _Translation mode_ is set to **Main windows** or **Notifications** 

**Steps to reproduce**

**Expected behavior**

**Screenshots**

**Environment**

- OS: 
- Desktop environment: 
- Crow Translate version: 

**Additional context**

**Comment by leonid1812** [comment]
version: **2.11.1-win64**

**Comment by leonid1812** [comment]
Ctrl+Alt+E work fine if **Translation mode** is set to **Main windows** or **Notifications**

**Comment by Shatur** [comment]
Can't reproduce on my side...

**Comment by leonid1812** [comment]
> Can't reproduce on my side...

version 2.10.7 - works fine

**Comment by leonid1812** [comment]
https://github.com/crow-translate/crow-translate/assets/17238147/3f6f8186-d287-413d-9776-93c8499f08e2

**Comment by Shatur** [comment]
Does this happen in all applications?

**Comment by leonid1812** [comment]
Yes, I’m experiencing this issue in all applications. I’ve also closed all running apps to exclude any hotkey conflicts, but the result remains the same. It seems that Crow Translate recognizes that I pressed _Ctrl+Alt+E_, but no window with translation appears.

I suppose this happened with version 2.10.8

**Comment by Shatur** [comment]
Stange, it works for me on Windows 10. No sure why it have problems on the latest Windows version.
